src/main/kotlin/org/opensearch/commons/alerting/model/Monitor.kt

```diff
@@ -332,6 +338,11 @@ data class Monitor(
                     } else {
                         xcp.booleanValue()
                     }
+                    SHOULD_PERSIST_FINDINGS_AND_ALERTS_FIELD -> delegateMonitor = if (xcp.currentToken() == XContentParser.Token.VALUE_NULL) {
```

Comment:
thinking of a scenario where this field is null
shouldn't we persist alerts and findings by default but we have initilalized `var delegateMonitor = false` and wont end up persisting

Reply:
same as https://github.com/opensearch-project/common-utils/pull/757#discussion_r1872129866